.github/workflows/maven.yml

```diff
@@ -3,24 +3,29 @@
 
 name: Java CI with Maven
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push, pull_request]
```

Comment:
so here we run  it regardless of the branch, correct?

Reply:
Yes - might be neat for feature branches without PR to have it.